index.js

```diff
@@ -48,6 +48,14 @@ const inputs = [
         argument: 'repository'
     }),
     read({
+        prompt: 'Jekyll enable, type true if yes',
+        argument: 'jekyll'
+    }),
+    read({
+        prompt: 'Jekyll path, e.g. site, press Enter if no Jekyll',
+        argument: 'sitepath'
```

Comment:
The `jekyll` and `sitepath` options should be optional - yarg's `default` option can be used for this: https://github.com/yargs/yargs#defaultkey-value-description